velox/common/base/BitUtil.h

```diff
@@ -355,9 +356,17 @@ void forEachBit(
         if (!word) {
```

Comment:
Do we really need this branch? the `while(word)` below will already exit immediately if word is zero.